test/instrumentation/modules/express.js

```diff
@@ -10,7 +10,6 @@ var http = require('http')
 
 var express = require('express')
 var test = require('tape')
-
```

Comment:
Could you add this blank line back please? We have an unwritten "rule" in our code style where node modules from npm are separated from internal modules by a blank line 🙂

Reply:
ok, It seems that another rule failed.
```javascript
agent.captureError = function (err, data) {}
```
// change to
```javascript
agent.captureError = function () {}
```
I have modified it